### DISCUSSION TITLE
Does GraphQL Hive support login via Okta portal?

### DISCUSSION THREAD

**Discussion by cookieMr** [graphql-hive/console#426] [Help] [answered]
The company i work for is interested in GraphQL Hive (cload hosted) and the 1st question we asked ourselves is *Does GraphQL Hive supports login via Okta?* Does it (assuming the company i work for goes with a paid plan)?

**Comment by n1ru4l** [comment]
It does not currenlty, but SuperTokens folks ensured us that it is definitely possible to implement this. We have not yet prioritized this yet, but definitely something we want to support in the future!

If you want to help us out, let us know! A good starting point will probably be to reach out on the SuperTokens Discord server: https://supertokens.com/

**Reply by cookieMr** [reply]
Thx @n1ru4l for this answer. Keep up the good work and stay awesome :)

**Reply by n1ru4l** [reply]
@cookieMr Would you expect only [OCID SSO](https://developer.okta.com/docs/concepts/oauth-openid/) or also [SCIM](https://developer.okta.com/docs/concepts/scim/) capabilities?

I am currently planning how we proceed with Okta and it seems that we need to submit Hive to the [Okta Integration Network (OIN)](https://developer.okta.com/docs/guides/okta-integration-network/) in order to allow multiple organizations to log in via their Okta organizations.

**Reply by cookieMr** [reply]
Hi @n1ru4l,

Big thx for picking this task up!
From what i learned today the `OCID SSO` would be enough but don't rely on just 1 user response. There might be others with other needs.

**Reply by cookieMr** [reply]
Took me a while but i got an answer from a person who knows much more about Okta in my company:
> [...] We recommend a tool that is SAML/OIDC or SCIM enabled to ensure we have end-user provisioning/de-provisioning function. [...]

**Reply by n1ru4l** [reply]
This is now implemented https://www.the-guild.dev/blog/announcing-free-single-sign-on-for-graphql-hive

**Comment by n1ru4l** [comment, marked as answer]
This is now implemented for both Cloud and self-hosted GraphQL Hive https://www.the-guild.dev/blog/announcing-free-single-sign-on-for-graphql-hive
